testing/test_gl_surface.cc

```diff
@@ -83,10 +85,26 @@ static std::string GetEGLError() {
 
 TestGLSurface::TestGLSurface(SkISize surface_size)
     : surface_size_(surface_size) {
-  display_ = ::eglGetDisplay(EGL_DEFAULT_DISPLAY);
+  PFNEGLGETPLATFORMDISPLAYEXTPROC egl_get_platform_display_EXT =
+      reinterpret_cast<PFNEGLGETPLATFORMDISPLAYEXTPROC>(
+          eglGetProcAddress("eglGetPlatformDisplayEXT"));
+  FML_CHECK(egl_get_platform_display_EXT)
+      << "eglGetPlatformDisplayEXT not available.";
+
+  const EGLint display_config[] = {
+      EGL_PLATFORM_ANGLE_TYPE_ANGLE,
+      EGL_PLATFORM_ANGLE_TYPE_VULKAN_ANGLE,
+      EGL_PLATFORM_ANGLE_DEVICE_TYPE_ANGLE,
+      EGL_PLATFORM_ANGLE_DEVICE_TYPE_SWIFTSHADER_ANGLE,
+      EGL_NONE,
+  };
+
+  display_ = egl_get_platform_display_EXT(
+      EGL_PLATFORM_ANGLE_ANGLE, reinterpret_cast<void*>(EGL_DEFAULT_DISPLAY),
```

Comment:
Does this actually work? I would have expected this to be `::eglGetDisplay(EGL_DEFAULT_DISPLAY)`

Reply:
Yes. But maybe it's cheating? I can use the get display call

Reply:
I have to do it this way because of https://bugs.chromium.org/p/angleproject/issues/detail?id=7435

Added a comment with a link to that bug.

Reply:
Per the ANGLE folks this is actually expected behavior - the eglGetPlatformDisplayEXT call expects a native display pointer, which is not what is returned by `eglGetDisplay`.